glfw/build/root.build

```diff
@@ -13,14 +14,15 @@ test.target = $c.target
 tcls = $c.target.class
 tsys = $c.target.system
 
-config [bool] config.glfw.usewayland ?= false
-config [bool] config.glfw.useosmesa ?= false
-config [bool] config.glfw.vulkan ?= false
+config [bool] config.glfw.usewayland ?= true
+config [bool] config.glfw.usex11 ?= true
 config [bool] config.glfw.usehybridhpg ?= false
 
-useosmesa = (($tcls == 'linux' || $tsys == 'freebsd') && $config.glfw.useosmesa)
-usewayland = ($tcls == 'linux' && $config.glfw.usewayland == true && !$config.glfw.useosmesa)
-usex11 = ($tcls == 'linux' && !$config.glfw.usewayland && !$config.glfw.useosmesa)
+linux = ($tcls == 'linux')
+unix  = ($tcls == 'freebsd' || $tcls == 'linux')
```

Comment:
```suggestion
unix = ($tcls == 'freebsd' || $tcls == 'linux')
```

Reply:
Is one extra space worth the alignment of four very similar lines? :pensive: (jk)